src/SprykerEco/Zed/Computop/Business/Hook/ComputopPostSaveHook.php

```diff
@@ -67,6 +63,7 @@ public function execute(QuoteTransfer $quoteTransfer, CheckoutResponseTransfer $
      */
     protected function setRedirect(QuoteTransfer $quoteTransfer, CheckoutResponseTransfer $checkoutResponseTransfer)
     {
+        $quoteTransfer->setOrderReference($checkoutResponseTransfer->getSaveOrder()->getOrderReference());
```

Comment:
why ?
Looks like this value is never used

Reply:
It's used to send orderReference to Computop.